src/pkg.jl

```diff
@@ -0,0 +1,218 @@
+using Pkg
+using Pkg: TOML
+
+const manifest_memoize = Ref{Dict{String, Any}}()
+
+function current_manifest()
+    if !isassigned(manifest_memoize)
+        manifest_memoize[] = TOML.parsefile(replace(Base.active_project(), "Project.toml" => "Manifest.toml"))
```

Comment:
Project files can be called `JuliaProject.toml` as well.

Reply:
Can also use `Pkg.Types.Context().env.manifest`

Reply:
That returns a dict with quite a different format... I think I'll keep it this way for now, and we can make Pkg improvements in another PR ;) I'm sure you have quite a few more improvements!